test/cases/parsing/issue-2050/x.js

```diff
@@ -1,8 +1,8 @@
 import { xa, xb, xc, xd } from "./module";
 
-module.exports = {
-	xa: xa,
-	xb: xb,
-	xc: xc,
-	xd: xd
-}
\ No newline at end of file
+export {
+	xa,
+	xb,
+	xc as xc,
+	xd as xd
+}
```

Comment:
`export { xa, xb, xc as xc, xd as xd } from "./module"`?

or is this specifically checking `export` of `import`ed symbol?